packages/sdk-react/src/components/products/ProductDetails/ProductEditForm/ProductEditForm.test.tsx

```diff
@@ -51,13 +43,13 @@ describe('ProductEditForm', () => {
         />
       );
 
-      await waitUntilTableIsLoaded();
-
-      const saveButton = screen.getByRole('button', {
+      const saveButton = screen.findByRole('button', {
         name: /cancel/i,
       });
 
-      fireEvent.click(saveButton);
+      await expect(saveButton).resolves.not.toBeDisabled();
```

Comment:
Generally the same

Reply:
Yeah, just thinking of a new approach. As long as it's a valid Jest/Vite syntax. The way specified in Notion is only a proposal and has not been adopted by the Monite Team, if I'm not confused.